ca/config.go

```diff
@@ -482,24 +496,9 @@ func LoadTLSCreds(rootCA RootCA, paths CertPaths) (*MutableTLSCreds, *MutableTLS
 
 	// Now that we know this certificate is valid, create a TLS Certificate for our
 	// credentials
-	var (
-		keyPair tls.Certificate
-		newErr  error
-	)
-	keyPair, err = tls.X509KeyPair(cert, key)
+	keyPair, err := tls.X509KeyPair(cert, key)
 	if err != nil {
-		// This current keypair isn't valid. It's possible we crashed before we
-		// overwrote the current key. Let's try loading it from disk.
```

Comment:
I remember this code was written because the cert and key are separate files and therefore are not written as one atomic unit. If we crash in between writing them, the key and cert won't be consistent - thus this two-phase logic that can look at a temporary key file if the normal key file doesn't match. Are there other changes that make this unnecessary?